x-pack/test/functional_embedded/tests/iframe_embedded.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import Url from 'url';
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['security', 'common']);
+  const browser = getService('browser');
+  const config = getService('config');
+  const testSubjects = getService('testSubjects');
+
+  describe('in iframe', () => {
+    it('should open Kibana for logged-in user', async () => {
+      await PageObjects.security.login();
+
+      const { protocol, hostname, port } = config.get('servers.kibana');
+
+      const url = Url.format({
+        protocol,
+        hostname,
+        port,
+        pathname: 'iframe_embedded',
+      });
+
+      await browser.navigateTo(url);
+
+      const iframe = await testSubjects.find('iframe_embedded');
+      await browser.switchToFrame(iframe);
+
+      await testSubjects.find('kibanaChrome', 60000);
```

Comment:
Is the `kibanaChrome` (from `src/core/public/rendering/rendering_service.tsx` I guess) really not displayed when on the login page / when unauthenticated ?

Reply:
I'd expect this to be hidden on the `chromeless` app on the `/login` page. That's a surprise!

Reply:
We also had to resort code like [this](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/page_objects/security_page.ts#L100-L103) to make sure chrome is present, but it's not a chromeless app :)

Reply:
@azasypkin I switched to `PageObjects.common.isChromeHidden();` instead.

Reply:
Yeah, I can't recall why, but it didn't work for me (was either too flaky or there was some other reason I cannot remember now :see_no_evil: )

Reply:
@azasypkin Probably you dealt with `isChromeVisible`, which didn't work out for me either since it finds `top-nav` which is not present in DOM (tested on master as well).

Reply:
Ah, yeah, maybe that was it. That's weird that they don't complement each other..